packages/node/src/plugins/CommonJsChunkLoadingPlugin.ts

```diff
@@ -12,14 +12,15 @@ interface CommonJsChunkLoadingOptions extends ModuleFederationPluginOptions {
   remotes: Record<string, string>;
   name?: string;
   asyncChunkLoading: boolean;
+  verbose?: boolean;
 }
 
 class CommonJsChunkLoadingPlugin {
   private options: CommonJsChunkLoadingOptions;
   private _asyncChunkLoading: boolean;
 
   constructor(options: CommonJsChunkLoadingOptions) {
-    this.options = options || ({} as CommonJsChunkLoadingOptions);
+    this.options = {...(options || ({} as CommonJsChunkLoadingOptions)), verbose: options?.verbose ?? false};
```

Comment:
can you set default value to options.verbose instead?

Reply:
ah yeah, good idea

Reply:
I think I've understood what you've asked for here, but if not let me know